tagsbox/react-instantsearch/src/lib/Tags.css

```diff
@@ -0,0 +1,77 @@
+.ais-Highlight-highlighted {
+  font-weight: bold;
+  text-decoration: underline; }
+
+.ais-TagsBox {
```

Comment:
looks like the compiled css here can also be removed

Reply:
This one is used in the Component, not the .scss one :)